src/pages/HomePage.tsx

```diff
@@ -27,8 +28,8 @@ function HomePage() {
 
   const fetchHouses = async () => {
     await handleAsyncAction(async () => {
-      const fetchedHouses = await readAll('houses');
-      setHouses(() => fetchedHouses);
+      const resp = await axios.get<BasicHouseResponseType>('http://localhost:4000/api/houses');
```

Comment:
It would be best if the host part of the URL was taken from environment variables and the URL was constructed with the use of constants in a separate file and imported as a constant to this file. The file with URL constants could be named something like `api.ts`, `server.ts` or `back(-)end.ts` and possibly placed in the folder `urls`/`URLs` directly inside `src`. The convention for naming such constants could be `x_ENDPOINT` so inside let's say in `server.ts`, there could be something like this:
```
const API_URL = `${import.meta.env.SERVER_HOST_NAME}/api`

const HOUSES_ENDPOINT = API_URL + "/houses"
```